### PR TITLE
NFDIV-515: connections field added to ccd definitions

### PR DESCRIPTION
**Pull request by SarahLittlejohn**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/NFDIV-515

### Change description ###
connections field added to ccd definitions
ccd-config-generator updated to latest version

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ X ] No
```

### CODE REVIEW

File: ccd-definitions/build.gradle [hunk 1]
```diff
@@ -12,7 +12,7 @@ plugins {
   id 'checkstyle'
   id 'jacoco'
   id 'java'
-  id 'hmcts.ccd.sdk' version '0.15.2'
+  id 'hmcts.ccd.sdk' version '0.15.3'
```
Comment (nitinprabhu) [inline ccd-definitions/build.gradle:15]: you also need to bump `api/build.gradle`

**Review by nitinprabhu — DISMISSED** [review]

File: ccd-definitions/src/main/java/uk/gov/hmcts/reform/divorce/ccd/model/enums/JurisdictionConnections.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import uk.gov.hmcts.ccd.sdk.api.HasLabel;
+
+@Getter
+@AllArgsConstructor
+public enum JurisdictionConnections implements HasLabel {
+    @JsonProperty("A")
+    A("A"),
```
Comment (nitinprabhu) [inline ccd-definitions/src/main/java/uk/gov/hmcts/reform/divorce/ccd/model/enums/JurisdictionConnections.java:12]: does this have any specific meaning ? A means something B means something ? 
we can use ` @JsonPropertyDescription()` to add something to indicate what it means which may help if any one looks at the class

File: api/build.gradle [hunk 3]
```diff
@@ -6,7 +6,7 @@ plugins {
     id 'io.spring.dependency-management' version '1.0.11.RELEASE'
     id 'org.springframework.boot' version '2.4.3'
     id 'com.github.ben-manes.versions' version '0.38.0'
-    id 'hmcts.ccd.sdk' version '0.15.2'
+    id 'hmcts.ccd.sdk' version '0.15.3'
```
Comment (nitinprabhu) [inline api/build.gradle:9]: Also need to change versions in this file for `ccdConfigGenerator: '0.15.3'

File: ccd-definitions/src/main/java/uk/gov/hmcts/reform/divorce/ccd/model/enums/JurisdictionConnections.java (hunk 2 above)
Comment (mapingo) [reply ccd-definitions/src/main/java/uk/gov/hmcts/reform/divorce/ccd/model/enums/JurisdictionConnections.java:14]: Maybe the enum value could have a more explantory name for use in Java code.  Something like PET_RESP_RESIDENT("A")

**Review by nitinprabhu — APPROVED** [review]

**Pull request merged**